pkg/services/sqlstore/datasource.go

```diff
@@ -180,7 +180,7 @@ func UpdateDataSource(cmd *m.UpdateDataSourceCommand) error {
 			updateSession = sess.Where("id=? and org_id=?", ds.Id, ds.OrgId)
 		}
 
-		affected, err := updateSession.Update(ds)
+		affected, err := updateSession.AllCols().Omit("created").Update(ds)
```

Comment:
Sweet! Didnt know about AllCols :) 

All Torkel mentioned the sess.UseBool calls should be removed. Confusing to solve the same problem twice.

Reply:
Merging this once those removes have been removed.

Reply:
Nevermind. I made the change myself since its so small.